inc/vcf/validator.hpp

```diff
@@ -96,6 +97,11 @@ namespace ebi
        
       protected:
         virtual void parse_buffer(char const * p, char const * pe, char const * eof) = 0;
+
+        /**
+         * Previously seen records
+         */
+        RecordCache duplicates;
```

Comment:
If the cache doesn't store duplicates, wouldn't it be better to rename the variable to something like "previous_records"?